aips/passkeys.md

```diff
@@ -0,0 +1,504 @@
+
+- The asymmetric key type for the credential (`Ed25519`, `ECDSA`, `RSA`, etc.)
+- User verification requirements during assertions
+- Whether the WebAuthn credential should be bound to one device or available cross-platform. 
+
+It's important to carefully consider which options to select as they have significant impacts on the passkey user experience and **CANNOT** be reconfigured after credential creation.
+
+```
+dictionary PublicKeyCredentialCreationOptions {
+    required PublicKeyCredentialRpEntity              rp;
+    required PublicKeyCredentialUserEntity            user;
+
+    required BufferSource                             challenge;
+    required sequence<PublicKeyCredentialParameters>  pubKeyCredParams;
+
+    unsigned long                                     timeout;
+    sequence<PublicKeyCredentialDescriptor>           excludeCredentials = [];
+    AuthenticatorSelectionCriteria                    authenticatorSelection;
+    DOMString                                         attestation = "none";
+    AuthenticationExtensionsClientInputs              extensions;
+};
+```
+
+Some fields in `PublicKeyCredentialCreationOptions` have been highlighted below.
+
+**[`PublicKeyCredentialParameters`](https://www.w3.org/TR/webauthn-3/#dictdef-publickeycredentialparameters)**
+
+To make a WebAuthn credential compatible with Aptos, the `pubKeyCredParams` array should only contain supported signature schemes. At the present moment, only `secp256r1` is supported, which requires that the array have one element with `alg: -7` to denote `secp256r1` as the choice of credential type.
+
+**[`PublicKeyCredentialUserEntity`](https://www.w3.org/TR/webauthn-3/#dictdef-publickeycredentialuserentity)**
+
+Each authenticator stores a [credentials map](https://www.w3.org/TR/webauthn-3/#authenticator-credentials-map), a map from ([`rpId`](https://www.w3.org/TR/webauthn-3/#public-key-credential-source-rpid), [[`userHandle`](https://www.w3.org/TR/webauthn-3/#public-key-credential-source-userhandle)]) to public key credential source. If the user creates another credential with the same user handle as an existing credential on that authenticator, it will **OVERWRITE** the existing credential, thus overwriting the private key associated with a passkey account. To avoid this, it is **IMPERATIVE** for the relying party to maintain a list of previously registered credentials and corresponding user handles.
+ 
+**[`AuthenticatorSelectionCriteria`](https://www.w3.org/TR/webauthn-3/#dictdef-authenticatorselectioncriteria)**
+
+This field defines additional information about the credential and provides constraints that affect the UX of registering / asserting with the credential. See comments above the fields below for more info:
+
+```js
```

Comment:
This pseudocode and comments are not helpful. The AIP should translate this W3C standard gobbledygook for the reader & describe implications for the actual use case (Aptos/blockchains).

Reply:
Removed